lib/awful/widget/watch.lua

```diff
@@ -0,0 +1,84 @@
+---------------------------------------------------------------------------
+--- Watch widget.
```

Comment:
See https://github.com/awesomeWM/awesome/pull/486

> :D - do you have a link?

There is more, but the most recent is:
https://www.reddit.com/r/awesomewm/comments/3yd4wg/is_there_an_alternative_api_documentation/